docs/docsite/rst/dev_guide/style_guide/basic_rules.rst

```diff
@@ -41,6 +41,32 @@ Bad:
 Better:
     Extraction also requires large volumes of water.
 
+.. _one_sentence_per_line:
+
+Use one sentence per line
+-------------------------
+Consider using a technique called *one sentence per line* when composing paragraphs.
```

Comment:
```suggestion
Consider using a technique called *one sentence per line* when composing paragraphs, or use at least one line per sentence (following https://sembr.org/).
```

Reply:
@oraNod ^^ might address the option to follow sembr.. accept or reject and then we can merge this puppy!